docker/nginx/default.conf

```diff
@@ -0,0 +1,69 @@
+upstream flask {
+    server host.docker.internal:5000;
+}
+
+server {
+    listen 80;
+
+    server_name localhost;
+
+    gzip            on;
+    gzip_comp_level 6;
+    gzip_types      text/css application/javascript;
+
+    location / {
+        proxy_set_header Host $http_host;
+        proxy_set_header X-Script-Name $http_x_script_name/;
+        proxy_set_header X-Forwarded-For $remote_addr;
+        proxy_pass http://flask;
+    }
+
+    location /static {
+        expires 30d;
+        add_header Cache-Control "public";
+
+        proxy_set_header Host $http_host;
+        proxy_set_header X-Script-Name $http_x_script_name/;
+        proxy_read_timeout 60;
+        proxy_pass http://flask/static;
+    }
+
+    location /admin {
+        add_header Cache-Control "no-store, must-revalidate";
+        proxy_set_header Host $http_host;
+        proxy_set_header X-Script-Name $http_x_script_name/;
+        proxy_pass http://flask/admin;
+    }
+
+#     location /mapproxy {
+#         access_log /var/log/nginx/mapproxy.access.log;
+#         error_log /var/log/nginx/mapproxy.error.log;
+#
+#         proxy_pass http://127.0.0.1:xxxx;
+#         proxy_set_header Host $http_host;
+#         proxy_set_header X-Script-Name $http_x_script_name/mapproxy;
+#         proxy_read_timeout 300;
+#     }
+#
+#     location /osm-observer {
+#         access_log /var/log/nginx/osm-observer.access.log;
+#         error_log /var/log/nginx/osm-observer.error.log;
+#
+#         proxy_pass http://127.0.0.1:xxxx;
+#         proxy_set_header Host $http_host;
+#         proxy_set_header X-Script-Name $http_x_script_name/osm-observer;
+#         proxy_set_header X-Forwarded-Proto $scheme;
+#         proxy_read_timeout 300;
+#     }
+#
+#     location /svgserver {
+#         access_log /var/log/nginx/svgserver.access.log;
+#         error_log /var/log/nginx/svgserver.error.log;
+#
+#         proxy_pass http://127.0.0.1:xxxx;
+#         proxy_set_header Host $http_host;
+#         proxy_set_header X-Script-Name $http_x_script_name/svgserver;
+#         proxy_set_header X-Forwarded-Proto $scheme;
+#         proxy_read_timeout 300;
+#     }
+}
```

Comment:
I see now there is no ssl certificates involved. So you can forget the previous comment